build.sbt

```diff
@@ -2,8 +2,8 @@ import java.io.File
 import scala.reflect.io.Directory
 
 name := "rikai"
-crossScalaVersions := List("2.12.10", "2.13.7")
-scalaVersion := "2.12.10"
```

Comment:
the default scalaVersion should be 2.12.x

Because pyspark is default to using scala 2.12.x